### PR TITLE
fix(chat): redesign of empty conversation settings (Issue #2045, #2697, #1691)

### PR DESCRIPTION
**Pull request by IlyaBondar**
**Description:**

redesign of empty conversation settings

Issues:

- Issue #2045 
- Issue #2697
- Issue #1691

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2657.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2657.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2657.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2657.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2657.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2657.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2657.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2657.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat/src/components/Chat/EmptyChatDescription.tsx [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+        }),
+      );
+    },
+    [conversation.id, dispatch],
+  );
+
+  return (
+    <div className="flex size-full flex-col items-center p-0 md:px-5 md:pt-5">
+      <div className="flex size-full flex-col items-center gap-px rounded">
+        {!modelsLoaded ? (
```
Comment (Alexander-Kezik) [inline apps/chat/src/components/Chat/EmptyChatDescription.tsx:79]: What about `View` component?
And move huge part of logic inside of it, like versions grouping, etc.

Comment (Alexander-Kezik) [reply apps/chat/src/components/Chat/EmptyChatDescription.tsx:79]: decided to do it in fix/2651-hide-empty-conversation branch

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2657.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by Alexander-Kezik — APPROVED** [review]

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2657.nightly-test.deltixhub.io
>E2E tests status: success

**Review by irinakartun — APPROVED** [review]

**Pull request merged**